### PR TITLE
kubernetes: add 1.0.1-1.9.3 docs

### PR DESCRIPTION
**Pull request by jimmidyson**
- [ ] Blocker 
- [ ] High
- [x] Medium

### CODE REVIEW

File: pages/services/kubernetes/1.0.0-1.9.3/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: Kubernetes 1.0.0-1.9.3
```
Comment (MatApple) [inline pages/services/kubernetes/1.0.0-1.9.3/index.md]: @jimmidyson should this file be carried over to the new directory and do we need to update `navigationTitle` & `title`?

Comment (jimmidyson) [reply pages/services/kubernetes/1.0.0-1.9.3/index.md]: This remains where it was, there's a new index file for the new version - https://github.com/mesosphere/dcos-docs-site/pull/708/files#diff-1118b5ae3f30c34b4df74ed8a9c791c4

Comment (jimmidyson) [reply pages/services/kubernetes/1.0.0-1.9.3/index.md]: It's a bit confusing. This file was only updated to change the weighting which affects ordering in the versions grid page at https://docs.mesosphere.com/services/kubernetes/ (look at other services for example of multiple versions available on that level of page).

**Comment by jimmidyson** [comment]
Closing this... going to be a 1.0.1-1.9.4 release instead to pull in a security fix from Kubernetes.

**Pull request closed without merging**